src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -502,6 +502,11 @@ private function addServiceMethodCalls($id, Definition $definition, $variableNam
     private function addServiceOverriddenMethods($id, Definition $definition)
     {
         $class = $this->container->getReflectionClass($definition->getClass());
+
+        if (!$class) {
+            throw new RuntimeException(sprintf('Unable to configure service "%s": class could not be reflected.', $id));
```

Comment:
Hum, I was thinking that maybe "could not be reflected" would be better, if there are other reasons than "not finding the class". But I can change it.

I'll check where the other calls are made.